### DISCUSSION TITLE
macOS scripted deployment

### DISCUSSION THREAD

**Discussion by franton** [CrowdStrike/falcon-scripts#241] [Ideas]
ChrisB on the mac admins slack suggested I post this.

I wrote this script a while back. Relies on API access to programmatically obtain the correct release of crowdstrike falcon before installing it on a mac using a management tool like Jamf Pro. There's other scripts in there.

Githhub: https://github.com/franton/Crowdstrike-API-Scripts
Companion blog post: https://richard-purves.com/2022/05/03/downloading-crowdstrike-via-api-for-fun-and-profit/

I use all of these on a daily basis.

It's all MIT license so feel free to use as you wish as long as proper attribution is given.

**Comment by carlosmmatos** [comment]
@franton Appreciate this a lot! Will be taking a look at adding support for macOS early next year

**Comment by JamesLochheadCRWD** [comment]
I'd find this really useful myself @carlosmmatos. The ability to setup MacOS hosts in CI/CD would be really useful for Services. 

The tricky part is installing the Systems Extensions. That is going to be quite difficult.

https://falcon.crowdstrike.com/documentation/page/e261a9b7/falcon-sensor-for-mac-deployment#t007715b
